src/WebSocket/WebSocketDispatcher.ts

```diff
@@ -0,0 +1,114 @@
+/*
+*  Send and receive requests serially
+*  next reuqest is only sent when the previous one got its response
+*/
+
+import { Dispatcher } from '../api';
+
+const errorMessages = {
```

Comment:
better to create a separate file for errors

Reply:
There is no need to export/import those values